depends/packages.md

```diff
@@ -152,16 +152,9 @@ Most autotools projects can be properly staged using:
 
 ## Build outputs:
 
-In general, the output of a depends package should not contain any libtool
-archives. Instead, the package should output `.pc` (`pkg-config`) files where
+In general, the package should output `.pc` (`pkg-config`) files where
```

Comment:
I don't think this is correct. Given we still have depends packages that can output libtool archives (which we currently remove).

Reply:
Thanks! Reverted back.